pkg/ddl/db_change_failpoints_test.go

```diff
@@ -36,14 +36,14 @@ import (
 
 // TestModifyColumnTypeArgs test job raw args won't be updated when error occurs in `updateVersionAndTableInfo`.
 func TestModifyColumnTypeArgs(t *testing.T) {
+	store := testkit.CreateMockStore(t)
```

Comment:
Is there any reason to move those 2 lines?

Reply:
I moved these two lines because the following failpoint will make `DROP TABLE` in bootstrap fail.

I just realized that I should just remove the `CREATE table mysql.schema_index_usage` in bootstrap, rather than create and drop it 🤦 . Now I modified to not create the table, so that this test won't need to be changed.

(The `drop` in upgradation is still needed, but it'll not affect the test).